pkg/parser/model/ddl.go

```diff
@@ -558,6 +566,10 @@ type Job struct {
 
 	// CDCWriteSource indicates the source of CDC write.
 	CDCWriteSource uint64 `json:"cdc_write_source"`
+
+	// LocalMode indicates whether the job is running in local TiDB.
+	// Only happens when tidb_ddl_version = 2.
+	LocalMode bool
```

Comment:
No need to marshal? How do I know whether a history job is executed in local mode?

Reply:
3ee5717

Reply:
[f7cd234](https://github.com/pingcap/tidb/pull/49687/commits/f7cd234326907db36bbfceddf2d82ebb705b62f4)